src/Steps/Release/PlanRelease.php

```diff
@@ -404,45 +404,78 @@ protected function reviewLibraryVersion(
         InputInterface $input,
         LibraryRelease $selectedVersion
     ) {
-        $question = new Question(
-            "Please enter a new version to release for <info>"
-            . $selectedVersion->getLibrary()->getName() . "</info>: ",
-            $selectedVersion->getVersion()
-        );
-        $newVersionName = $this->getQuestionHelper()->ask($input, $output, $question);
+        $versions = [
+            'new_version' => [
+                'text' => 'Please enter a new version to release for <info>%s</info>: ',
+                'default' => $selectedVersion->getVersion(),
+            ],
+            'prior_version' => [
+                'text' => 'Optional: modify the prior version for <info>%s</info>: ',
+                'default' => $selectedVersion->getPriorVersion(),
+            ],
+        ];
+
+        foreach ($versions as $key => $options) {
+            $question = new Question(
+                sprintf($options['text'], $selectedVersion->getLibrary()->getName()),
+                $options['default']
+            );
+            $result = $this->getQuestionHelper()->ask($input, $output, $question);
+
+            // If version is invalid, show an error message
+            if (!Version::parse($result)) {
+                $this->log(
+                    $output,
+                    "Invalid version {$result}; Please enter a tag in w.x.y(-[rc|alpha|beta][z]) format",
+                    "error"
+                );
+                $this->reviewLibraryVersion($output, $input, $selectedVersion);
+                return;
+            }
 
-        // If version is valid, update and return
-        if (Version::parse($newVersionName)) {
-            // Warn if upgrade-only and selected version isn't an existing tag
-            if ($selectedVersion->getLibrary()->isUpgradeOnly()) {
+            // Warn if upgrade-only OR specifying a previous version and selected version isn't an existing tag
+            $upgradeOnly = $selectedVersion->getLibrary()->isUpgradeOnly();
+            if ($upgradeOnly || $key === 'prior_version') {
                 $tags = $selectedVersion->getLibrary()->getTags();
-                if (!array_key_exists($newVersionName, $tags)) {
+                if (!array_key_exists($result, $tags)) {
```

Comment:
This could be null if the user provides no input?

Reply:
Weirdly it actually returns a Version object. I've added some logic above to handle that